preprod.healdata.org/portal/gitops.json

```diff
@@ -118,6 +118,7 @@
     "analysis": false,
     "discovery": true,
     "discoveryUseAggMDS": true,
+    "workspaceTokenServiceRefreshTokenAtLogin": true,
```

Comment:
should also enable `discoveryUseAggWTS` in here

Reply:
Thank you for pointing that feature flag @mfshao, I have addressed that in the latest commit